dimod/binary/binary_quadratic_model.py

```diff
@@ -624,6 +624,84 @@ def add_linear_equality_constraint(
                     u, v, 2 * lagrange_multiplier * ubias * vbias)
         self.offset += lagrange_multiplier * constant * constant
 
+    def add_linear_inequality_constraint(
+            self, terms: Iterable[Tuple[Variable, Bias]],
+            lagrange_multiplier: Bias, constant: Bias,
+            label: str,
+            slack_method: str = "log2",
+            slack_range: int = None,
+            discontinuous_slack: bool = False
+    ):
+        """Add a linear inequality constraint as a quadratic objective.
+
+        Adds a linear inequality constraint of the form
+        :math:`\\sum_{i} a_{i} x_{i} + C <= 0`
+        to the binary quadratic model as a quadratic objective. Only integral coefficient are handled correctly.
+        Try multiplying both sides of equations by factor of 10s to remove fractional coefficients.
+
+        Args:
+            terms (iterable/iterator):
+                An iterable of 2-tuples, (variable, bias).
+                Each tuple is evaluated to the term (bias * variable).
+                All terms in the list are summed.
+            lagrange_multiplier:
```

Comment:
We often use this name for input parameters like these but @mcfarljm has pointed out that in many cases that's inaccurate and should be something like ``penalty_weight`` or similar.

Reply:
Should we also fix this in the  `add_linear_equality_constraint`? I'd like to stay consistent with the docstring we have for that method.

Reply:
Changing it in ` add_linear_equality_constraint` would be a backwards compatibility break, so we should keep them consistent.